runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessCollectionTile/ProcessCollectionTile.styles.ts

```diff
@@ -0,0 +1,71 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Paper, Typography } from '@mui/material';
+import styled from 'styled-components';
+
+export const ProcessCollectionTileWrapper = styled(Paper)(({ theme }) => `
+    &[class*="MuiPaper"] {
+        display: flex;
+        align-items: center;
+        max-width: 350px;
+        gap: 8px;
+        padding: 2px 0 2px 20px;
+        background-color: ${theme.palette.grey[200]};
+    }
+`);
+
+export const CollectionNameWrapper = styled.div`
+    max-width: 230px;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+`;
+
+export const MenuWrapper = styled.div`
+    margin-left: auto;
+`;
+
+export const CollectionListWrapper = styled.div<{ isExpanded?: boolean }>`
+    height: ${({ isExpanded }) => isExpanded ? '100%' : '50px'};
+    overflow: hidden;
+`;
+
+export const ExpandButton = styled.div
+    <{ $isExpanded: boolean; }>(({ theme, $isExpanded }) => `
```

Comment:
check the devtools - probably you'll get a warning saying you should use lowercase, for the props in elementary html elements

the solution is to e.g. change isExpanded -> expanded